docs/tutorials/basic/ruby.md

```diff
@@ -153,7 +153,7 @@ a special gRPC Ruby plugin.
 
 If you want to run this yourself, make sure you've installed protoc and followed
 the gRPC Ruby plugin [installation
-instructions](https://github.com/grpc/grpc/blob/{{ site.data.config.grpc_release_tag }}/src/cpp/README.md#make) first):
+instructions](https://github.com/grpc/grpc/blob/{{ site.data.config.grpc_release_tag }}/src/ruby/README.md) first):
```

Comment:
Acually, even this looks likes it must be a type, this part is about installing protoc and grpc_ruby_plugin for protoc and https://github.com/grpc/grpc/blob/master/src/ruby/README.md doesn't mention how that's supposed to be done for ruby.

@apolcyn  should the README.md (and/or this document) mention the usage of "grpc-tools" package? https://rubygems.org/gems/grpc-tools/versions/1.18.0

Reply:
yeah, you're right. I'll merge this and open a follow-up

Reply:
actually, I don't have write-access for this repo, so I can't merge